ExploreASL_Master.m

```diff
@@ -29,14 +29,22 @@
 %   x              - Struct containing pipeline environment parameters, useful when only initializing ExploreASL/debugging
 %                         
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
-% DESCRIPTION:    This masterscript starts ExploreASL by first calling ExploreASL_Initialize, 
-%                 then running xASL_Module_Structure, xASL_module_ASL and xASL_module_Population.
-%                 When ProcessModules is set to 0, the ExploreASL pipeline is not started but only  
-%                 initialized for debugging. This pipeline can be run from CLI or using the python GUI.
+% DESCRIPTION:               This master script runs the following six modular workflows:
 % 
-% ExploreASL_Initialize    - This wrapper initializes ExploreASL: managing paths, deployment, etc.
-% ExploreASL_ImportMaster  - Multi-step import workflow for DCM2NII, NII2BIDS & BIDS2LEGACY.
-% ExploreASL_ProcessMaster - Multi-step processing workflow for the STRUCTURAL, ASL and POPULATION module.
+% I. Initialization:         ExploreASL_Initialize

```

Comment:
Not a huge issue of course. But is there any reason to have Roman instead Arabic numbers?

Reply:
@jan-petr: I talked with Henk about that. We thought it could be helpful if we do not use arabic numbers for the import and process modules which are themselves part of the ExploreASL_Master workflow. Alternatively we could use A, B, C, D etc.

Reply:
Now we have:

- `I.` Initialization
- `II.` Import Master
    - `1.` DICOM TO NII
    - `2.` NII TO BIDS
    - `3.` DEFACE
    - `4.` BIDS TO Legacy
- `III.` Data loading
- `IV.` Print user feedback (before pipeline)
- `V.` Processing Master
    - `1.` Structural Module
    - `2.` ASL Module
    - `3.` Population Module
- `VI.` Print user feedback (after pipeline)

Reply:
Very nice!